x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/empty_prompt/empty_prompt.tsx

```diff
@@ -63,7 +63,7 @@ export const AnalyticsEmptyPrompt: FC<Props> = ({ disabled, onCreateFirstJobClic
           <p>
             <FormattedMessage
               id="xpack.ml.overview.analyticsList.emptyPromptText"
-              defaultMessage="Data frame analytics enables you to perform outlier detection, regression, or classification analysis and put the annotated data in a new index. The classification and regression trained models can also be used for inference in pipelines and aggregations."
+              defaultMessage="Perform outlier detection, regression, or classification analysis and put the annotated data in a new index. You can also use trained models for inference in pipelines and aggregations."
```

Comment:
This is nice and short and snappy. I wonder if we need to make it clearer that we generate the trained models as part of this process. For example, switch from "use" to "create":
```suggestion
              defaultMessage="Perform outlier detection, regression, or classification analysis and put the annotated data in a new index. Create trained models to add inference in pipelines and aggregations."
```